boards/demo_f030c8.json

```diff
@@ -0,0 +1,38 @@
+{
+  "build": {
+    "cpu": "cortex-m0",
+    "extra_flags": "-DSTM32F030x8",
+    "f_cpu": "48000000L",
+    "mcu": "stm32f030c8t6",
+    "variant": "DEMO_F030C8"
+  },
+  "debug": {
+    "default_tools": [
+      "stlink"
+    ],
+    "jlink_device": "STM32F030C8",
+    "onboard_tools": [
+      "stlink"
+    ],
+    "openocd_target": "stm32f0x",
+    "svd_path": "STM32F030.svd"
+  },
+  "frameworks": [
+    "arduino",
+    "mbed",

```

Comment:
Are you sure this board supports mbed framework? What variant?

Reply:
No, Mbed framework does not support it yet. So remove it too.